### DISCUSSION TITLE
How to figure out why my issue form template is not applied?

### DISCUSSION THREAD

**Discussion by karfau** [community/community#5025] [Projects and Issues] [answered]
I just created a new repository to test the issue forms templates: https://github.com/karfau/experiments/blob/85a8772ced58d94f2a7df6cd42d4c42cef910b40/.github/ISSUE_TEMPLATE/NEW-BOOK.yml
I even configured `blank_issues_enabled: false` in [config.yml](https://github.com/karfau/experiments/blob/main/.github/ISSUE_TEMPLATE/config.yml)
but when clicking the "New issue" it is still creating a blank issue.

How can I learn/understand why the template is not being applied?
Is there a tool to validate my template yml against? 
Are there any logs that I can inspect to see what is going on?

When maintaining a template and introducing an issue, you will suddenly get blank issues again, so is there some way to check for it, e.g. in github acitons on a branch?

**Comment by karfau** [comment]
Would I have opened the link myself I would have seen the hint provided by github:
![image](https://user-images.githubusercontent.com/135657/129106468-d235c9cf-86c9-4985-bbd7-c66dd870682b.png)
So my first and third questions were answered, but the one about maintenance and tooling are still open.

**Comment by lukehefson** [comment, marked as answer]
> Is there a tool to validate my template yml against?

Hey @karfau. There are various online YAML validation tools you might like to look into – although we don’t include one ‘natively’ as part of the issue form building experience. I noticed [in this thread](https://github.com/github/feedback/discussions/4907) that a user debugged using http://www.yamllint.com.
